gui-tk/wiring.tcl

```diff
@@ -235,7 +235,7 @@ bind .wiring.canvas <Button-5> {zoomAt  %x %y [expr 1.0/1.1]}
 bind .wiring.canvas <MouseWheel> { if {%D>=0} {zoomAt %x %y 1.1} {zoomAt  %x %y [expr 1.0/(1.1)]} }
 
 if {[tk windowingsystem]=="aqua"} {
-    bind .wiring.canvas <Command-Button-1> {
+    bind .wiring.canvas <Meta-Button-1> {
         tk_messageBox -message "Mouse coordinates %x %y"
```

Comment:
Maybe we'll have to forgo the mouse coordinate reporting function on Macs. It's only for debugging purposes anyway.

FYI, the Meta modifier only works on certain historical keyboards  - eg the Sun keyboard had a meta key IIRC, with a diamond symbol on it. I only ever used it access emacs "meta" functions(eg meta-x). On PC keyboards, emacs interpreted Alt as the meta modifier, and on Mac keyboards, it interprets Command as such (or maybe "Apple", I forget - my last Apple keyboard would be now more than 20 years old, and all the labels worn off).

Maybe I should just take this ticket over, and test out the Mac pathways through this script.

Reply:
> Maybe we'll have to forgo the mouse coordinate reporting function on Macs. It's only for debugging purposes anyway.
> 
> FYI, the Meta modifier only works on certain historical keyboards - eg the Sun keyboard had a meta key IIRC, with a diamond symbol on it. I only ever used it access emacs "meta" functions(eg meta-x). On PC keyboards, emacs interpreted Alt as the meta modifier, and on Mac keyboards, it interprets Command as such (or maybe "Apple", I forget - my last Apple keyboard would be now more than 20 years old, and all the labels worn off).
> 


I see what I've done wrong now. The $meta option confused me, because I thought it was defined as "set meta Control" for the keyboard shortcuts. Hence, I didn't understand that it could be used on Macs as command or control. I saw the Meta modifier in Tcl's documentation and thought you meant that instead...

> Maybe I should just take this ticket over, and test out the Mac pathways through this script.

That might be a good idea, thanks. I have no experience with Macs keyboards and because I'm using keyboards with a Spanish layout, which sometimes caused me headaches on Windows machines, it makes things all the more confusing for me.